[2024-01-08T09:19:58.612+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: fetching_cleaning.Fetch_from_Postgresql manual__2024-01-08T09:19:57.535563+00:00 [queued]>
[2024-01-08T09:19:58.616+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: fetching_cleaning.Fetch_from_Postgresql manual__2024-01-08T09:19:57.535563+00:00 [queued]>
[2024-01-08T09:19:58.616+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2024-01-08T09:19:58.617+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 2
[2024-01-08T09:19:58.617+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2024-01-08T09:19:58.623+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): Fetch_from_Postgresql> on 2024-01-08 09:19:57.535563+00:00
[2024-01-08T09:19:58.630+0000] {standard_task_runner.py:52} INFO - Started process 780 to run task
[2024-01-08T09:19:58.637+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'fetching_cleaning', 'Fetch_from_Postgresql', 'manual__2024-01-08T09:19:57.535563+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmpusqamj20', '--error-file', '/tmp/tmpqluxn5xr']
[2024-01-08T09:19:58.640+0000] {standard_task_runner.py:80} INFO - Job 13: Subtask Fetch_from_Postgresql
[2024-01-08T09:19:58.710+0000] {task_command.py:371} INFO - Running <TaskInstance: fetching_cleaning.Fetch_from_Postgresql manual__2024-01-08T09:19:57.535563+00:00 [running]> on host 13eafc15112b
[2024-01-08T09:19:58.763+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=saepul
AIRFLOW_CTX_DAG_ID=fetching_cleaning
AIRFLOW_CTX_TASK_ID=Fetch_from_Postgresql
AIRFLOW_CTX_EXECUTION_DATE=2024-01-08T09:19:57.535563+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-01-08T09:19:57.535563+00:00
[2024-01-08T09:19:58.901+0000] {python.py:173} INFO - Done. Returned value was: None
[2024-01-08T09:19:58.919+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=fetching_cleaning, task_id=Fetch_from_Postgresql, execution_date=20240108T091957, start_date=20240108T091958, end_date=20240108T091958
[2024-01-08T09:19:58.971+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-01-08T09:19:58.997+0000] {local_task_job.py:279} INFO - 1 downstream tasks scheduled from follow-on schedule check
